lib/rubocop/cop/vendor/rollbar_inside_rescue.rb

```diff
@@ -29,7 +29,7 @@ module Vendor
       #     end
       #   end
       #
-      class RollbarInsideRescue < Cop
+      class RollbarInsideRescue < Base
```

Comment:
curious, why was this change needed?

Reply:
RuboCop has a rule that enforces this. I don't fully know why because the comment in the code links to a page that 404s. But I think that RuboCop::Cop::Base is meant to be easier to work with.

https://github.com/rubocop/rubocop/blob/4fcf0ffb942b1d37e646e4230e7f1a8802feb9ca/lib/rubocop/cop/cop.rb#L8-L10

Reply:
Ah, my comment got lost because I rebased and force pushed. Posting here as well in case anyone is curious:

RuboCop has a rule that enforces this. I don't fully know why because the comment in the code links to a page that 404s. But I think that RuboCop::Cop::Base is meant to be easier to work with.

https://github.com/rubocop/rubocop/blob/4fcf0ffb942b1d37e646e4230e7f1a8802feb9ca/lib/rubocop/cop/cop.rb#L8-L10